game-core/src/main/java/games/strategy/engine/auto/health/check/LocalSystemChecker.java

```diff
@@ -84,6 +83,6 @@ private static SystemCheck defaultFileSystemCheck() {
   private String getStatusMessage() {
     return systemChecks.stream()
         .map(SystemCheck::getResultMessage)
-        .collect(Collectors.joining("\n"));
+        .collect(Collectors.joining(System.lineSeparator()));
```

Comment:
Not sure.  I switched this in order to be consistent with the use of `%n` in the `String#format()` call a few lines above (i.e. to have a consistent line separator to replace in the `textToHtml()` code).  Since this text also goes to the console, I suppose it's possible the use of `\n` could lead to incorrect formatting on Windows (e.g. the legacy Notepad effect).

I really wish Java would adopt a _universal newlines_ mode like Python so this wouldn't even be an issue. :disappointed: